src/Widget/Form.php

```diff
@@ -50,7 +51,7 @@ public function start(): string
                 if (($pos1 = strpos($pair, '=')) !== false) {
                     $hiddenInputs[] = Html::hiddenInput(
                         urldecode(substr($pair, 0, $pos1)),
-                        urldecode(substr($pair, $pos1 + 1))
+                        urldecode((string) substr($pair, $pos1 + 1))
```

Comment:
While it theoretically can return `false`, I see it as excessive type-casting in this and alike cases.